### DISCUSSION TITLE
RSS feeds and ruTorrent labels disappear when server dies

### DISCUSSION THREAD

**Discussion by jakobbg** [Novik/ruTorrent#2489] [Q&A]
I do not have a UPS on my system, and sometimes the server experiences a power outage. Consequence is that all RSS feeds and all torrent labels disappear from ruTorrent.

Is it possible to ensure I do not lose all labels and other config? It is quite the pain to rebuild all that label metadata and to set up the RSS feeds again every time I lose power on my system.

Of course, a UPS would not hurt either…

Sysinfo: FreeBSD 13.2. Running latest ruTorrent from git.

**Comment by stickz** [comment]
Is this happening on version 4.0.4 as well? Or just version 4.1? There were some changes made to RSS and Labels in version 4.1. We just need to know where to look. Another possibly is a version change issue.

**Comment by jakobbg** [comment]
Hard to debug later on. Will try to find a reproducible case. Thanks.
